packages/subgraph/tests/dao/utils.ts

```diff
@@ -128,7 +128,7 @@ export function createNewGrantedEvent(
     ethereum.Value.fromAddress(Address.fromString(oracle))
   );
 
-  newGrantedEvent.parameters.push(roleParam);
+  newGrantedEvent.parameters.push(contractPermissionIDParam);
```

Comment:
really a minor comment, but I'd suggest `permissionId` with lowercase `d` (can be ignored here)

Reply:
Applied lowercase `d` everywhere